doc/source/train/getting-started-transformers.rst

```diff
@@ -1,14 +1,14 @@
 .. _train-pytorch-transformers:
 
-Getting Started with Hugging Face Transformers
-==============================================
+Get Started with Hugging Face Transformers
```

Comment:
Should the doc titles all start with a verb?

Reply:
For these action based docs, we do. Are you seeing some that don't, or do you have a concern?

Reply:
I see. Just want to confirm so that I can follow this manner in the future.